src/main/java/com/example/event_manager/service/TaskStatusServiceImpl.java

```diff
@@ -0,0 +1,26 @@
+package com.example.event_manager.service;
+
+import com.example.event_manager.model.TaskStatus;
+
+import java.util.List;
+
+public class TaskStatusServiceImpl implements TaskStatusService {
+  @Override
+  public boolean save(TaskStatus taskStatus) {
+    return false;
+  }
+
+  @Override
+  public void delete(Long id) {
+  }
```

Comment:
why it does nothing? add a comment if it suppose to be like there or throw NotImplementedException

Reply:
if it throws then thow this exception in method implementation - in other case it make no sense to add throws to signature